### PR TITLE
Upcoming Git course

### PR DESCRIPTION
**Pull request by ns-rse**

### CODE REVIEW

File: _events/workshop-2023-05-30-git-zero-hero.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Persistent failure to attend booked courses might result in you being excluded from future training opportunities.
+
+<hr/>
+
+More details and registration via
+[eventbrite](https://www.eventbrite.co.uk/e/git-github-through-gitkraken-from-zero-to-hero-registration-626271082667).
+
+
+**Please note:** This workshop will run over 2 half days from 09:30-13:00 on Monday 6th and Tuesday 7th March 2023.
```
Comment (Robadob) [inline _events/workshop-2023-05-30-git-zero-hero.md]: Haven't updated the date here, still refers to March.

Comment (ptheywood) [inline _events/workshop-2023-05-30-git-zero-hero.md]: Incorrect dates (in the past)

```suggestion
**Please note:** This workshop will run over 2 half days from 09:30-13:00 on Tuesday 30th May and Thursday 1st June 2023.
```

Dates based on eventbrite info:

> + 2023-05-30 09:30 to 13:00
> + 2023-06-01 09:30 to 13:00

File: _events/workshop-2023-05-30-git-zero-hero.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+*This event is for University of Sheffield researchers and research students only. Please register with an
+**@shef.ac.uk** or **@sheffield.ac.uk** email address to ensure your ticket request is accepted.*
+
+
+**Prerequisite skills:** Some programming experience.
+
+**Learning Objectives**
+* Version controlling your own project through Git & GitHub.
```
Comment (ptheywood) [inline _events/workshop-2023-05-30-git-zero-hero.md:28]: Might need a blank line before the list start for this to render correctly? Might just be my markdown linter being fussy.

**Review by ptheywood — CHANGES_REQUESTED** [review]

**Comment by ns-rse** [comment]
Thank you both, apologies for missing the dates at the bottom.

**Review by ptheywood — APPROVED** [review]

**Pull request merged**